.github/workflows/main.yml

```diff
@@ -194,6 +194,10 @@ jobs:
       with:
         name: engine-release
 
+    - name: Cache build
+      uses: Swatinem/rust-cache@v1.4.0
```

Comment:
Note: Nice! This will be the first time that we use a third-party action in our workflows which makes me think of a few questions about this change
Do we know what our policy is on using third-party actions?
We do have an [.github/actions](https://github.com/fastly/js-compute-runtime/tree/main/.github/actions) folder where we stored our own custom actions, should we be vendoring this third-party action into that folder?
Or maybe we can use the github cache action and configure it for rust, [following the example](https://github.com/actions/cache/blob/main/examples.md#rust---cargo)?

Reply:
These are good points! I think I'll use the github cache action like you mentioned, as that'll avoid any questions about policy or vendoring for now :)